### DISCUSSION TITLE
503 when two fediverse instances communicate with each other

### DISCUSSION THREAD

**Discussion by Tealk** [NginxProxyManager/nginx-proxy-manager#2059] [Q&A]
Hello all,

I have a Mastodon and a Mobilizon instance on 2 V servers and these behind the Nginx Proxy Manager. Now when I want to access `https://rollenspiel.social/api/v2/search?q=@schnupperrunde@rollenspiel.events&resolve=true&limit=5` the Mobilizon instance from the Mastodon instance I get a 503 
But from other instances outside my network I can access each of the two instances.
Does anyone have an idea where I could start?

**Comment by valon0388** [comment]
Hi. Did you ever get this error resolved? I'm trying to set up a self-hosted instance of Mastodon and having some trouble with the reverse proxy config. How do you have yours set up? I'll try and help troubleshoot if needed.

**Reply by Tealk** [reply]
I have assigned the internal ips to the domains via dns
